libs/shared/ui/chart/src/lib/chart/chart.component.html

```diff
@@ -1,5 +1,5 @@
 <google-chart
-  *ngIf="data"
+  *ngIf="data$ | async as chartData"
```

Comment:
data property is not defined in component. Also for cleaner and safer subscriptions, better to consider async pipe in template.